### DISCUSSION TITLE
Firebase Authentciation Not Working After Build

### DISCUSSION THREAD

**Discussion by acsonmez** [community/community#127552] [Programming Help]
### Select Topic Area

Question

### Body

Hello,
I made a game for oculus in Unity. I'm using Firebase Authentication. You can log in with your e-mail and password.
But my problem is this: Everything works fine in Unity. After building my game to my VR Headset, the login process does not work. There is no way to log in.
I tried many methods on the web but I couldn't figure it out.
Maybe anyone has an idea? Can you help me?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @acsonmez ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
